src/Faker/Provider/lt_LT/Person.php

```diff
@@ -0,0 +1,151 @@
+<?php
+
+namespace Faker\Provider\lt_LT;
+
+use Faker\Calculator\Luhn;
+
+class Person extends \Faker\Provider\Person
+{
+    protected static $maleNameFormats = array(
+        '{{firstNameMale}} {{lastNameMale}}',
+        '{{firstNameMale}} {{lastNameMale}}',
+        '{{firstNameMale}} {{lastNameMale}}',
+        '{{firstNameMale}} {{lastNameMale}}',
+        '{{titleMale}} {{firstNameMale}} {{lastNameMale}}',
+        '{{titleMale}} {{firstNameMale}} {{lastNameMale}}',
+        '{{firstNameMale}} {{lastNameMale}}',
+    );
+
+    protected static $femaleNameFormats = array(
+        '{{firstNameFemale}} {{lastNameFemale}}',
+        '{{firstNameFemale}} {{lastNameFemale}}',
+        '{{firstNameFemale}} {{lastNameFemale}}',
+        '{{firstNameFemale}} {{lastNameFemale}}',
+        '{{titleFemale}} {{firstNameFemale}} {{lastNameFemale}}',
+        '{{firstNameFemale}} {{lastNameFemale}}',
+        '{{titleFemale}} {{firstNameFemale}} {{lastNameFemale}}',
+    );
+
+    protected static $firstNameFemale = array(
+        "Aliūnė", "Kantvyda", "Mykalina", "Galkantė", "Rolana", "Dorina", "Dalija",
+        "Mudrė", "Gražytė", "Michasė", "Vincentė", "Prantiška", "Liudviga", "Mėlana", "Miroleta",
+        "Kangailė", "Jeugenija", "Palagija", "Anida", "Rigina", "Daivutė", "Sandora", "Marka", "Childa",
+        "Ksavera", "Raima", "Alvirutė", "Alita", "Uolė", "Leonova", "Marija", "Eitė", "Elfrosina",
+        "Eigė", "Liuzija", "Polkarija", "Erdvila", "Dolita", "Elza", "Vilgelma", "Drovydė", "Gintenė",
+        "Tariotė", "Ougenija", "Yda", "Agripina", "Liamutė", "Emilja", "Enata", "Vladimera", "Ula", "Alfaga",
+        "Vijolita", "Veranika", "Georgina", "Germinė", "Dzidra", "Egona", "Burkanta", "Ritma", "Fedosija",
+        "Josifa", "Ramytė", "Vilgarda", "Vėjūnė", "Kamita", "Iva", "Gedgailė", "Marijina", "Sofi", "Auksana",
+        "Almonė", "Benidikta", "Norminta", "Kirmantė", "Juozapava", "Vitorija", "Domincelė", "Rusana", "Lucyna",
+        "Kamilija", "Vientautė", "Etmyra", "Barutė", "Kantminė", "Sanita", "Agotė", "Dangirda", "Daugotė",
+        "Vaišviltė", "Nerutė", "Kostė", "Burtautė", "Mintara", "Gaudvyda", "Alga", "Elinora", "Iliminata",
+        "Aegina", "Idilija"
+    );
+
+    /**
+     * Source: From my memory
```

Comment:
Nah, impossible to maintain. Please find an online, open, and serious datasource.